src/main/java/domain/card/Card.java

```diff
@@ -0,0 +1,46 @@
+package domain.card;
+
+import java.util.Objects;
+
+public class Card {
+    private final Denomination denomination;
+    private final Suit suit;
+
+    public Card(Denomination denomination, Suit suit) {
+        this.denomination = denomination;
+        this.suit = suit;
+    }
+
+    public boolean isAceCard() {
+        return this.denomination.equals(Denomination.ACE);
```

Comment:
```suggestion
        return this.denomination == Denomination.ACE;
```
enum은 == 비교가 가능하답니다. equals를 사용했을때보다, ==을 사용했을 때 장점 [참고자료](https://chan9.tistory.com/174) 공유합니다.